multicluster/controllers/multicluster/memberclusterannounce_controller.go

```diff
@@ -72,10 +72,12 @@ type MemberClusterAnnounceReconciler struct {
 }
 
 type MemberClusterStatusManager interface {
-	AddMember(MemberId common.ClusterID)
-	RemoveMember(MemberId common.ClusterID)
+	AddMember(memberID common.ClusterID)
+	RemoveMember(memberID common.ClusterID)
 
 	GetMemberClusterStatuses() []multiclusterv1alpha1.ClusterStatus
+	// This is for unit test only
+	GetTimerData() map[common.ClusterID]*timerData
```

Comment:
had similar comments before, does test code really need this interface to access the data?

Reply:
oh, yes, it's unnecessary, removed it.